io.github.mandruis7.xbox-cloud-gaming-electron.yaml

```diff
@@ -0,0 +1,64 @@
+app-id: io.github.mandruis7.xbox-cloud-gaming-electron
+base: org.electronjs.Electron2.BaseApp
+base-version: '22.08'
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node16
+command: start-xbox-cloud-gaming-electron.sh
+separate-locales: false
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --share=network
+  - --socket=pulseaudio
+  - --socket=x11
+  - --socket=system-bus
```

Comment:
this is not allowed.

Reply:
I can potentially remove the --socket=system-bus just QA what effects has on the app

Reply:
not *potentially*. Individual dbus name required should be added as needed.

Reply:
Just removed the line